<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>106</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>86</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLastEdited">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>15</y>
     <width>166</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>LastEdited</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>291</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ProgramName</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
